tests/test_itk_torch_bridge.py

```diff
@@ -0,0 +1,440 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import os
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.apps import download_url
+from monai.data import ITKReader
+from monai.data.itk_torch_bridge import (
+    get_itk_image_center,
+    itk_image_to_metatensor,
+    itk_to_monai_affine,
+    monai_to_itk_affine,
+    monai_to_itk_ddf,
+)
+from monai.networks.blocks import Warp
+from monai.transforms import Affine
+from monai.utils import optional_import
+from tests.utils import skip_if_downloading_fails, testing_data_config
+
+itk, has_itk = optional_import("itk")
+
+TESTS = ["CT_2D_head_fixed.mha", "CT_2D_head_moving.mha", "copd1_highres_INSP_STD_COPD_img.nii.gz"]
```

Comment:
I've uploaded the test cases to 
- https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/CT_2D_head_fixed.mha
- https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/CT_2D_head_moving.mha

could you please update the data_config.json and let the tests download them on the fly? thanks!